NEWS.md

```diff
@@ -1,5 +1,8 @@
 # dtplyr (development version)
 
+*    `group_by()` now has a `key` parameter which, if set to `FALSE`, sets the 
```

Comment:
What about `reorder`? Or some thing that speaks directly to the action, rather than the underlying data.table implementation?

Reply:
`reorder` works, but I feel like it could be misunderstood ("reorder from what?"). To follow the dplyr naming convention, I chose `arrange` instead. That should, in theory, be more understandable for people familiar with the tidyverse.